### DISCUSSION TITLE
Checkstyle workflow

### DISCUSSION THREAD

**Discussion by aayushbrahmbhatt** [aayushbrahmbhatt/Simplii#7] [Announcements]
Hey team if anyone is available please finish up setting up the checkstyle plugin in the workflows as it will be make code more readable thank you! and ping me after someone finishes it up! cheers!!

**Comment by Tanay27102002** [comment]
working on it Aayush!!

**Comment by Tanay27102002** [comment]
What are the plugins that I should use?

**Comment by aayushbrahmbhatt** [comment]
Flake8: Pylint: Black: isort: mypy: try any one of these @Tanay27102002

**Comment by naisargshah** [comment]
Guys I am working on my test cases so is anyone of you free to give me some help?
